### ISSUE TITLE
Error on startup for svelte app

### ISSUE THREAD

**Issue by Keyruu** [authorizerdev/authorizer#208]
**Version:** 0.39.0

**Describe the bug**

Importing authorizer-js throws error:
```
Uncaught Error: Module "stream" has been externalized for browser compatibility. Cannot access "stream.Readable" in client code.
    at Object.get (browser-external:stream:9:13)
    at index.mjs:11:25
```

**Steps To Reproduce**

npm init vite

select svelte

yarn

yarn add @authorizerdev/authorizer-js

add "import { Authorizer } from '@authorizerdev/authorizer-js'" to App.svelte or any other file

**Desktop (please complete the following information):**

- OS: macOS
- Browser chrome

**Comment by lakhansamani** [comment]
@Keyruu thank you for reporting it, will look into it.

**Comment by lakhansamani** [comment]
@Keyruu I was checking on this and found that `vite` has this problem
https://stackoverflow.com/a/72453533

Might have to replace `node-fetch` with `isomorphic-fetch`

Will do and that and let you know.

**Comment by lakhansamani** [comment]
@Keyruu issue is fixed in `@authorizerdev/authorizer-js@0.16.1`
